packages/patternfly-4/react-charts/src/components/ChartZoomContainer/ChartZoomContainer.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import hoistNonReactStatics from 'hoist-non-react-statics';
+import { VictoryZoomContainer } from 'victory';
+import { merge } from 'lodash';
```

Comment:
Where are we picking up the lodash package? I don't see it in the react-core dependencies.

Reply:
@jcaianirh  - I don't believe we currently use lodash in react-charts.  We generally prefer pure es6/typescript methods instead of shelling out to a library for things like this - can you try to use native functionality instead?